src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +48,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() <= toAge
+                        && person.getAge() >= fromAge
+                        && person.getSex().equals(Person.Sex.MAN))
```

Comment:
It's generally better to call .equals() on enum when we have such opportunity as we're sure it has proper value
```suggestion
                        && Person.Sex.MAN.equals(person.getSex())
```